README.rdoc

```diff
@@ -1,28 +1,21 @@
-== README
-
-This README would normally document whatever steps are necessary to get the
-application up and running.
-
-Things you may want to cover:
-
-* Ruby version
-
-* System dependencies
-
-* Configuration
-
-* Database creation
-
-* Database initialization
-
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
-
-
-Please feel free to use a different markup language if you do not plan to run
-<tt>rake doc:app</tt>.
+Tentacle
+======
+Tentacle creates the pipes to sell anywhere on the web, w/ inventory synced. Sell through a tweet or a photo...it's up to you.
+Product Roadmap
+------
+Features currently under construction include:
+1. Login
```

Comment:
Please change to 'Account creation' as this reflects more than simply logging in. Otherwise 💯